docs/src/man/getting_started.md

```diff
@@ -306,7 +306,7 @@ julia> DataFrame(a = 1:2, b = [1.0, missing],
                  c = categorical('a':'b'), d = [1//2, missing])
 2×4 DataFrame
 │ Row │ a     │ b        │ c            │ d         │
-│     │ Int64 │ Float64⍰ │ Categorical… │ Rationa…⍰ │
+│     │ Int64 │ Float64? │ Categorical… │ Rationa…? │
```

Comment:
While we're breaking things, maybe worth also revisiting the printing of `Categorical` so that users don't have to adjust doctests in the next release again? We discussed printing just `Cat` to save some space.

Reply:
OK - changed. I will hold off with merging until you have a look and approve the change from `Categorical` to `Cat`.

(there are extensive changes to the manual because of this change - and you will see there the visual consequences of the change to assess it)